spot_msgs/action/NavigateTo.action

```diff
@@ -1,7 +1,4 @@
-string upload_path # Absolute path to map_directory, which is downloaded from tablet controller
-string navigate_to # Waypoint id string for where to go
-bool initial_localization_fiducial   # Tells the initializer whether to use fiducials
-string initial_localization_waypoint # Waypoint id to trigger localization 
+string id_navigate_to # Waypoint id string for where to go
```

Comment:
This seems your original idea, beccause no one have noticed the necessarity of update interface https://github.com/heuristicus/spot_ros/blob/master/spot_msgs/action/NavigateTo.action

Whta it the benefit of this change? To support any new features ? https://dev.bostondynamics.com/docs/release_notes#autowalk-service ?

Reply:
It does not related to any new features. Original spot-ros API limits the capability of AutoWalk API since it loads all the graph files when any navigation command is issued even if the same graph is already loaded to the robot. It extremely lengthens the time it takes to execute a single navigation command.

Reply:
In fact, there are some other people use this update https://github.com/heuristicus/spot_ros/pull/50